jujugui/static/gui/src/app/assets/css/inspector/_juju-inspector.scss

```diff
@@ -295,29 +314,52 @@
                 }
             }
         }
-        .destroy-service-prompt,
+
         .controls {
             $transition: .25s, max-height .3s;
```

Comment:
Looks like this var is only used once. It should probably be removed and add the value to the transition include below.